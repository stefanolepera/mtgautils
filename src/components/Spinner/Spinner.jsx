import React from 'react';
import styled, { keyframes } from 'styled-components';

const spin = keyframes`
    0% { transform: rotate(0deg); }
    100% { transform: rotate(360deg); }
`;

// const StyledSpinner = styled.div`
//     margin: 40px auto;
//     border: 5px solid #ccc;
//     border-top: 5px solid maroon;
//     border-radius: 50%;
//     width: 50px;
//     height: 50px;
//     animation: ${spin} 1s linear infinite;
// `;

const StyledSpinner = styled.div`
    margin: 10px auto;
    border: 5px solid #ccc;
    border-top: 5px solid maroon;
    border-radius: 50%;
    width: 30px;
    height: 30px;
    animation: ${spin} 1s linear infinite;
`;

const Spinner = () => <StyledSpinner />;

export default Spinner;
