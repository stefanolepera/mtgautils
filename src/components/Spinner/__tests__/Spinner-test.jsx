import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import Spinner from '../Spinner';

describe('<ResultItem /> test', () => {
    it('Test snapshots of content', () => {
        const wrapper = shallow(<Spinner />);

        expect(shallowToJson(wrapper)).toMatchSnapshot();
    });
});
