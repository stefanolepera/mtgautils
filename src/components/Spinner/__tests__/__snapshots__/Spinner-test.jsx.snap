// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ResultItem /> test Test snapshots of content 1`] = `<styled.div />`;
